build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -266,9 +266,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="32,28,16,4,432,2556,2556,512,512,512,512,512,8,"
-CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,bin,bin,bin,bin,bootparam,"
-CONFIG_FLASH_PART_NAME="bl1,reserved,ftl,system_data,ss,kernel,kernel,userfs,app1,app1,app2,app2,bootparam,"
+CONFIG_FLASH_PART_SIZE="32,28,16,4,432,1408,1408,152,152,1508,1508,512,1024,8,"
```

Comment:
Let's use the same values as internal. 
```
1408,1408,1508,1508,152,152
```

Reply:
Since we have the following app sizes on public >
KERNEL    Binary Size :  1,295,328 bytes    Partition Size :  1,441,792 bytes
APP1    Binary Size :      9,529 bytes    Partition Size :    155,648 bytes
APP2    Binary Size :    284,215 bytes    Partition Size :  1,544,192 bytes

Lets keep it as it is.